unstructured/ingest/connector/chroma.py

```diff
@@ -0,0 +1,162 @@
+import itertools
+import json
+import typing as t
+import uuid
+from dataclasses import dataclass
+
+from unstructured.ingest.error import DestinationConnectionError
+from unstructured.ingest.interfaces import (
+    BaseConnectorConfig,
+    BaseDestinationConnector,
+    BaseIngestDoc,
+    WriteConfig,
+)
+from unstructured.ingest.logger import logger
+from unstructured.staging.base import flatten_dict
+from unstructured.utils import requires_dependencies
+
+if t.TYPE_CHECKING:
+    from chromadb import Collection as ChromaCollection
+
+    # from chromadb.config import Settings as ChromaSettings
+
+
+@dataclass
+class SimpleChromaConfig(BaseConnectorConfig):
+    collection_name: str
+    path: t.Optional[str] = None
+    settings: t.Optional[t.Dict[str, str]] = None
+    tenant: t.Optional[str] = None
+    database: t.Optional[str] = None
+    host: t.Optional[str] = None
+    port: t.Optional[int] = None
+    ssl: bool = False
+    headers: t.Optional[t.Dict[str, str]] = ({},)
```

Comment:
Should default to `None` if optional.
```suggestion
    headers: t.Optional[t.Dict[str, str]] = None
```